Builds/Docker/build-ci.sh

```diff
@@ -0,0 +1,13 @@
+set -e
+
+mkdir -p build/docker/
+cp doc/rippled-example.cfg build/clang.debug/rippled build/docker/
+cp Builds/Docker/Dockerfile-testnet build/docker/Dockerfile
+mv build/docker/rippled-example.cfg build/docker/rippled.cfg
+strip build/docker/rippled
+docker build -t ripple/rippled:$CIRCLE_SHA1 build/docker/
+docker tag ripple/rippled:$CIRCLE_SHA1 ripple/rippled:latest
```

Comment:
No, the underlying bug was the $CIRCLE_BRANCH variable resolving to 'pull/8375', which isn't a valid tag for docker. $CIRCLE_SHA1 though is always a sha1 hash which is fine.